src/sentry/monitors/validators.py

```diff
@@ -249,6 +250,12 @@ class MonitorValidator(CamelSnakeSerializer):
         default="active",
         help_text="Status of the monitor. Disabled monitors will not accept events and will not count towards the monitor quota.",
     )
+    owner = ActorField(
+        required=False,
+        allow_null=True,
+        help_text="The ID of the team or user that owns the monitor. (eg. user:51 or team:6)",
```

Comment:
Copied from here.

I wonder if we should update these to also have the examples?

https://github.com/getsentry/sentry/blob/bd6802b19bec14f08a38b89d3feb9dadae10e1e7/src/sentry/incidents/endpoints/organization_alert_rule_details.py#L283